packages/gatsby-plugin-utils/src/polyfill-remote-file/__tests__/gatsby-image-resolver.ts

```diff
@@ -76,6 +80,66 @@ describe(`gatsbyImageData`, () => {
     },
   }
 
+  it(`should return proper image props for aspect ratio and automatically add "fit" prop`, async () => {
+    const result = await gatsbyImageResolver(
+      portraitSource,
+      {
+        aspectRatio: 1.333333333,
+        layout: `fixed`,
+        width: 300,
+        placeholder: `none`,
+      },
+      actions
+    )
+
+    const parsedSrcSet = parseSrcSet(result.images.sources[0].srcSet)
+
+    expect(parsedSrcSet.length).toBe(2)
+
+    expect(parsedSrcSet[0].src).toEqual(
+      `/_gatsby/image/${base64Encode(portraitSource.url)}/${base64Encode(
+        `w=300&h=481&fit=crop&ar=1.333333333&fm=avif&q=75`
```

Comment:
we shouldn't pass arr here

Reply:
Can you explain the why? I don't think I fully follow :sweat_smile:

Reply:
Never mind, I figured it out already! :laughing:  We [use aspect ratio to calculate width / height](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-utils/src/polyfill-remote-file/graphql/gatsby-image-resolver.ts#L136-L139)